doc/polycli_monitor.md

```diff
@@ -30,6 +30,7 @@ If you're experiencing missing blocks, try adjusting the `--batch-size` and `--i
 ```bash
   -b, --batch-size string   Number of requests per batch (default "auto")
   -c, --cache-limit int     Number of cached blocks for the LRU block data structure (Min 100) (default 200)
+  -H, --header strings      Header to be added to each HTTP request. E.g. "X-First-Name: Joe"
```

Comment:
in same vein, consider adding bit extra clarity for users needing to pass multiple headers